snuba/datasets/metrics_bucket_processor.py

```diff
@@ -78,8 +82,13 @@ def _should_process(self, message: Mapping[str, Any]) -> bool:
 
     def _process_values(self, message: Mapping[str, Any]) -> Mapping[str, Any]:
         values = message["value"]
-        for v in values:
-            assert isinstance(v, int), "Illegal value in set. Int expected: {v}"
```

Comment:
These were probably meant to be f-strings and not regular strings so I updated them.